run.py

```diff
@@ -354,7 +296,7 @@ def start_neuro_san(self):
             sys.executable,
             "-u",
             "-m",
-            "neuro_san.service.main_loop.server_main_loop",
+            "servers.neuro_san.neuro_san_server_wrapper",
```

Comment:
The Python process that runs the Neuro SAN server must be initialized with the Phoenix stuff. We do that as a first step in the wrapper.
That allows us to remove `sitecustomize.py` that would initialize every single python process like `pip` or `pylint`. The only process that needs to know about Phoenix is the one that runs the Neuro SAN server.